Rakefile

```diff
@@ -84,239 +57,59 @@ desc "generate a release tarball"
 task :release => :compile do
```

Comment:
considering what you did with other long tasks, I think I'll extract that one too. For now, we can keep as it, there are already enough changed.